src/actions/index.js

```diff
@@ -45,28 +47,27 @@ const ALERT_LEVEL_ERROR = 'error'
 export const downloadFileMissing = () => ({ type: DOWNLOAD_FILE_E_MISSING, alert: { message: 'error.download_file.missing', level: ALERT_LEVEL_ERROR } })
 export const downloadFileOffline = () => ({ type: DOWNLOAD_FILE_E_OFFLINE, alert: { message: 'error.download_file.offline', level: ALERT_LEVEL_ERROR } })
 
-export const locationChange = (virtualRoot, folderId) => {
-  return async dispatch => {
-    dispatch({ type: LOCATION_CHANGE, virtualRoot, folderId })
-    return dispatch(openFolder(virtualRoot, folderId))
-  }
+export const openFiles = () => {
+  return async dispatch => dispatch(openFolder(ROOT_DIR_ID))
+}
```

Comment:
Cannot we write it like:

```js
export const openFiles = () => async (dispatch) => dispatch(openFolder(ROOT_DIR_ID))
```

or (if we don't want parenthesis for single parameter in arrow function, see my previous comment https://github.com/cozy/cozy-files-v3/pull/205/commits/f67437f801b54a346ab00eac603568b02e15cc99#r108860052)

```js
export const openFiles = () => async dispatch => dispatch(openFolder(ROOT_DIR_ID))
```